cou/steps/plan.py

```diff
@@ -48,7 +48,7 @@
 from cou.steps.backup import backup
 from cou.steps.hypervisor import HypervisorUpgradePlanner
 from cou.utils.app_utils import set_require_osd_release_option
-from cou.utils.juju_utils import DEFAULT_TIMEOUT, COUMachine
+from cou.utils.juju_utils import DEFAULT_TIMEOUT, COUMachine, COUUnit
```

Comment:
Not in the scope of this PR, but just want to point out `COUUnit` should be changed to `Unit`, like `Machine`, in a separate patch.

Reply:
That will be changed on main, and then we merge main to dev/data-plane with all changes we made.